src/java.base/share/classes/sun/security/x509/SerialNumber.java

```diff
@@ -100,9 +100,8 @@ public SerialNumber(InputStream in) throws IOException {
     /**
      * Return the SerialNumber as user readable string.
      */
-    public String toString() {
-        return "SerialNumber: [" + Debug.toHexString(serialNum) + ']';
-    }
+    public String toString() { return "Serial number: " +
+        serialNum.toString(16).toUpperCase().replaceAll("(?<=..)(..)", ":$1"); }
```

Comment:
I am not sure that `replaceAll` expression is correct. When I tried it for `BigInteger("74984748")`, it emitted "47:82:d2c" when I think it should be "04:78:2D:2C". I would use the HexFormat class here (and in `X509CertSelector`), ex:

`HexFormat.ofDelimiter(":").withUpperCase().formatHex(xcert.getSerialNumber().toByteArray())`